.github/workflows/release.yml

```diff
@@ -99,6 +99,52 @@ jobs:
         name: ${{ steps.artifact-name.outputs.name }}
         path: target/${{ matrix.target }}/release/astar-collator
 
+  evm-tracing-native-linux:
+    needs: checks-and-tests
+    runs-on: [self-hosted, Linux, X64]
+    strategy:
+      matrix:
+        chain: ["astar", "shiden", "shibuya"]
+    steps:
+
+    - name: Checkout the source code
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Install deps
+      run: sudo apt -y install protobuf-compiler
+
+    - name: x86_64 setup
+      run: |
+        mkdir -p .cargo
+        touch .cargo/config
+        printf '[target.x86_64-unknown-linux-gnu]\nrustflags = ["-Clink-arg=-fuse-ld=lld"]' >> .cargo/config
+
+    - name: Install & display rust toolchain
+      run: rustup show
+
+    - name: Check targets are installed correctly
+      run: rustup target list --installed
+
+    - name: Build optimized binary with evm tracing
+      run: CARGO_PROFILE_RELEASE_LTO=true RUSTFLAGS="-C codegen-units=1" cargo build --release --target x86_64-unknown-linux-gnu --features evm-tracing --verbose --locked
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: astar-ubuntu-tracing-latest
+        path: target/x86_64-unknown-linux-gnu/release/astar-collator
+
+    - name: rename tracing runtimes
+      run: |
+        cd target/x86_64-unknown-linux-gnu/wbuild/${{ matrix.chain }}-runtime
+        mv ${{ matrix.chain }}_runtime.compact.compressed.wasm ${{ matrix.chain }}_tracing_runtime.compact.compressed.wasm
```

Comment:
I suggest to omit `.compact.compressed` from the name but do append the spec version number (as in standard release).

Reply:
@Dinonard this one is just the artifact upload, I respected the same convention as it is for normal runtime built by `srtool`.
Renaming for release asset is done [here](https://github.com/AstarNetwork/Astar/blob/3d09c3a22fdce6cbe1b88bfbc4ee9c6604645449/.github/workflows/release.yml#L505).